Doc/c-api/conversion.rst

```diff
@@ -165,13 +165,41 @@ The following functions provide locale-independent string to number conversions.
 .. c:function:: int PyOS_stricmp(const char *s1, const char *s2)
 
    Case insensitive comparison of strings. The function works almost
-   identically to :c:func:`!strcmp` except that it ignores the case.
+   identically to :c:func:`!strcmp` except that it ignores the case and locale.
+
+   This is an alias of :c:func:`PyOS_mystricmp` on all platforms except Windows,
+   where it is an alias of :c:func:`!strcmp`.
```

Comment:
Watch out when working with C function names. You left out an `i` from the tongue-twisting mess, and got a very different function.
(same below)